projects/cashmere/src/lib/pipes/file-size/file-size.pipe.spec.ts

```diff
@@ -0,0 +1,57 @@
+import {async, inject, TestBed} from '@angular/core/testing';
+
+import { FileSizePipe } from "./file-size.pipe";
+
+describe("FileSizePipe", () => {
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
```

Comment:
Please convert these tests to not require `TestBed`.

Reply:
Removed test bed